docs/ops/detection/PriorBox_1.md

```diff
@@ -6,21 +6,55 @@
 
 **Short description**: *PriorBox* operation generates prior boxes of specified sizes and aspect ratios across all dimensions.
 
+**Detailed description**:
+
+*PriorBox* computes coordinates of prior boxes by following:
+1.  First calculates *center_x* and *center_y* of prior box:
+    \f[
+    W \equiv Width \quad Of \quad Image \\
+    H \equiv Height \quad Of \quad Image
+    \f]
+    *   If step equals 0:
+        \f[
+        center_x=(w+0.5) \\
+        center_y=(h+0.5)
+        \f]
+    *   else:
+        \f[
+        center_x=(w+offset)*step \\
+        center_y=(h+offset)*step \\
+        w \subset \left( 0, W \right ) \\
+        h \subset \left( 0, H \right )
+        \f]
+2.  Then, for each \f$ s \subset \left( 0, min\_sizes \right ) \f$ calculates coordinates of prior boxes:
+    \f[
+    xmin = \frac{\frac{center_x - s}{2}}{W}
+    \f]
+    \f[
+    ymin = \frac{\frac{center_y - s}{2}}{H}
+    \f]
+    \f[
+    xmax = \frac{\frac{center_x + s}{2}}{W}
+    \f]
+    \f[
+    ymin = \frac{\frac{center_y + s}{2}}{H}
+    \f]
```

Comment:
There is no mentioning of `clip` attribute. I expect to see it in the detailed description.

Reply:
I don't see how `aspect_ratio` and `flip` attributes are used as well

Reply:
Made sure to mention `clip`, however I don't think that it's necessary to provide pseudo-code for the whole operation. Reference implementation is quite readable for that purpose.